tests/test_integration.py

```diff
@@ -12,9 +13,24 @@
     AutoEncodingTopicModel,
     ClusteringTopicModel,
     KeyNMF,
-    SemanticSignalSeparation,
+    SemanticSignalSeparation
```

Comment:
why? I think Black added it btw. Are you using black?

Reply:
yup, sry - linted now